### ISSUE TITLE
"Bill" valure required on split transaction

### ISSUE THREAD

**Issue by psychowood** [firefly-iii/firefly-iii#3841]
In v5.4.1 the Bill = (none) value is ignored/considered not valid, you cannot save a split transaction without adding a Bill.

<img width="1784" alt="image" src="https://user-images.githubusercontent.com/115389/94045709-170d6480-fdd0-11ea-845b-25fc44bf4e28.png">

**Comment by JC5** [comment]
Can't replicate, stay tuned.

**Comment by JC5** [comment]
Still can't replicate. Could you share how you did this?

**Comment by psychowood** [comment]
I tested it a little bit more, it seems to happen only when editing a transaction and not when creating a new one.

- Edit a transaction for example [this one](https://demo.firefly-iii.org/transactions/edit/460)
- Click Add Another Split
- Click Update Transaction

It is the same whether you input some information or not.

**Comment by JC5** [comment]
Thanks, found it. :+1:

**Comment by psychowood** [comment]
I'm not sure if this is related to your latest commit, but creating a split transaction from scratch on the demo site does not show the group, only the splits, that are not editable.

<img width="1496" alt="image" src="https://user-images.githubusercontent.com/115389/94074512-b5fa8680-fdf9-11ea-88aa-e9c3d74929ea.png">

Btw I just noticed on the screenshot that the two transactions were assigned a category _new-category_ and the tag _some,new,tag_ but it probably was the work of a rule, it's not happening on my installations.

Transaction [here on the demo site](https://demo.firefly-iii.org/transactions/show/461), I built the link manually.

On my local instance it works correctly:

<img width="1481" alt="image" src="https://user-images.githubusercontent.com/115389/94075058-c2cbaa00-fdfa-11ea-9709-bb4945150323.png">
